content/docs/user-guide/components/reference/atom/directional-light.md

```diff
@@ -13,8 +13,46 @@ The **Directional Light** component casts light from an infinitely distant point
 [Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/)
 
 
-## Properties
+## Directional Light Properties
 
-{{< todo issue=https://github.com/o3de/o3de.org/issues/717 >}}
-Add properties.
-{{< /todo >}}
+![Directional Light component interface.](/images/user-guide/components/reference/atom/light-component-ui/directional-light.png)
+
+| Property | Description | Values | Default |
+| - | - | - | - |
+| **Color** | The color of the light. The color acts as a mask or gel on perfectly white light source. For instance, a 1000 lux light with a white color will actually be 1000 lux, but a 1000 lux light with an 18% gray color will only be 180 lux. | Color | White |
+| **Intensity mode** | Allows specifying light values in lux or Ev100. Lux is a linear light value while EV100 is logarithmic (similar to camera stops). | `Lux` or `Ev100` | `Ev100` |
+| **Intensity** | Intensity of the light in the photometric unit set by **Intensity Mode**. | Any number for `EV100`, any positive number for `Lux` | `4.0` |
+| **Angular diameter** | The angular diameter of the light source in degrees. This is used to make sure specular highlights from directional lights don't completely disappear on mirror smooth surfaces. The angular diameter of the sun as viewed from earth is about 0.5°. | 0.0 - 5.0 | `0.5` |
+| **Shadow** | Refer to [Shadow properties](#shadow-properties) below. | | |
+| **Global Illumination** | Refer to [Global Illumination properties](#global-illumination-properties) below. | | |
+
+### Shadow properties
+
+| Property | Description | Values | Default |
+| - | - | - | - |
+| **Camera** | The entity of the camera the shadows will be visible from. Used to calculate the view frustums for shadow cascades. | Entity Id | None |
+| **Shadow far clip** | The distance to the far clip plane of the shadow. This is the maximum distance from the light source that shadows can cast. | 0 to Infinity | `100.0` |
+| **Shadowmap size** | The texture width and height of each shadow map cascade. | `256`, `512`, `1024`, `2048` | `1024` |
+| **Cascade count** | The number of shadow cascades. | 1 - 4 | `4` |
+| **Automatic splitting** | If enabled, the **Split ratio** is used, otherwise the **Far depth cascade** values are used. | Boolean | Enabled |

```

Comment:
```suggestion
| **Automatic splitting** | If enabled, the **Split Ratio** is used, otherwise the **Far depth cascade** values are used. | Boolean | Enabled |
```